tests/integrations/bedrock/bedrock_test.py

```diff
@@ -0,0 +1,107 @@
+import json
+
+import boto3
+import pytest
+
+import weave
+from weave.integrations.bedrock import patch_client
+
+model_id = "anthropic.claude-3-5-sonnet-20240620-v1:0"
+
+system_message = "You are an expert software engineer that knows a lot of programming. You prefer short answers."
+messages = [
+    {
+        "role": "user",
+        "content": [
+            {
+                "text": (
+                    "In Bash, how do I list all text files in the current directory "
+                    "(excluding subdirectories) that have been modified in the last month?"
+                )
+            }
+        ],
+    }
+]
+
+
+def _remove_body_from_response(response):
+    if "body" in response:
+        response["body"] = None  # Remove the body content
+    return response
+
+
+@pytest.mark.skip_clickhouse_client
+@pytest.mark.vcr(
+    filter_headers=[
+        "authorization",
+        "content-type",
+        "user-agent",
+        "x-amz-date",
+        "x-amz-security-token",
+        "x-amz-sso_bearer_token",
+        "amz-sdk-invocation-id",
+    ],
+    before_record_response=_remove_body_from_response,
+    allowed_hosts=[
+        "api.wandb.ai",
+        "localhost",
+        "portal.sso.us-east-2.amazonaws.com",
+        "bedrock-runtime.us-east-1.amazonaws.com",
+    ],
+)
+def test_bedrock_converse(client: weave.trace.weave_client.WeaveClient) -> None:
+    bedrock_client = boto3.client("bedrock-runtime")
+    patch_client(bedrock_client)
+
+    response = bedrock_client.converse(
+        modelId=model_id,
+        system=[{"text": system_message}],  # it needs a list for some reason
+        messages=messages,
+        inferenceConfig={"maxTokens": 30},
+    )
+
+    # Verify the response structure
+    assert response is not None
+    assert "output" in response
+    assert "message" in response["output"]
+    assert "content" in response["output"]["message"]
+
+
+@pytest.mark.skip_clickhouse_client
+@pytest.mark.vcr(
+    filter_headers=[
+        "authorization",
+        "content-type",
+        "user-agent",
+        "x-amz-date",
+        "x-amz-security-token",
+        "x-amz-sso_bearer_token",
+        "amz-sdk-invocation-id",
+    ],
+    allowed_hosts=["api.wandb.ai", "localhost"],
+)
+def test_bedrock_invoke(client: weave.trace.weave_client.WeaveClient) -> None:
+    bedrock_client = boto3.client("bedrock-runtime")
+    patch_client(bedrock_client)
```

Comment:
why are you doing patching in here?  Shouldn't this be done in `bedrock_sdk.py`?